ansible/roles/vm_set/library/vm_topology.py

```diff
@@ -1586,9 +1586,8 @@ def main():
             if vms_exists:
                 net.add_injected_fp_ports_to_docker()
                 net.bind_fp_ports()
-                if vm_type != "vsonic":
-                    net.bind_vm_backplane()
-                    net.add_bp_port_to_docker(ptf_bp_ip_addr, ptf_bp_ipv6_addr)
+                net.bind_vm_backplane()
```

Comment:
@Pterosaur, do you recollect why this check was added earlier ?

Reply:
Because the vsonic didn't support to inject bgp(exabgp) information from backplane before. I don't have related requirement so that I directly skip these code by the check. But I believe this PR is for fixing this issue.